CodeGen/benchmark/accuracy/README.md

```diff
@@ -1,73 +1,77 @@
 
 ## Evaluation Framework
 
-We evaluate accuracy by [bigcode-evaluation-harness](https://github.com/bigcode-project/bigcode-evaluation-harness). It is a framework for the evaluation of code generation models.
+We utilize the [bigcode-evaluation-harness](https://github.com/bigcode-project/bigcode-evaluation-harness), a framework specifically designed for evaluating code generation models. It supports various standard benchmarks such as [HumanEval](https://huggingface.co/datasets/openai_humaneval), [MBPP](https://huggingface.co/datasets/mbpp), and others.
 
-## Evaluation FAQs
+## Prerequisites
 
-### Launch CodeGen microservice
+- A running CodeGen service accessible via an HTTP endpoint. Refer to the main [CodeGen README](../../README.md) for deployment options.
+- Python 3.8+ environment.
+- Git installed.
 
-Please refer to [CodeGen Examples](https://github.com/opea-project/GenAIExamples/tree/main/CodeGen/README.md), follow the guide to deploy CodeGen megeservice.
+## Environment Setup
 
-Use `curl` command to test codegen service and ensure that it has started properly
+1.  **Clone the Evaluation Repository:**
 
-```bash
-export CODEGEN_ENDPOINT="http://${your_ip}:7778/v1/codegen"
-curl $CODEGEN_ENDPOINT \
-    -H "Content-Type: application/json" \
-    -d '{"messages": "Implement a high-level API for a TODO list application. The API takes as input an operation request and updates the TODO list in place. If the request is invalid, raise an exception."}'
+    ```shell
+    git clone https://github.com/opea-project/GenAIEval
+    cd GenAIEval
+    ```
 
-```
+2.  **Install Dependencies:**
+    ```shell
+    pip install -r requirements.txt
+    pip install -e .
+    ```
 
-### Generation and Evaluation
+## Running the Accuracy Benchmark
 
-For evaluating the models on coding tasks or specifically coding LLMs, we follow the [bigcode-evaluation-harness](https://github.com/bigcode-project/bigcode-evaluation-harness) and provide the command line usage and function call usage. [HumanEval](https://huggingface.co/datasets/openai_humaneval), [HumanEval+](https://huggingface.co/datasets/evalplus/humanevalplus), [InstructHumanEval](https://huggingface.co/datasets/codeparrot/instructhumaneval), [APPS](https://huggingface.co/datasets/codeparrot/apps), [MBPP](https://huggingface.co/datasets/mbpp), [MBPP+](https://huggingface.co/datasets/evalplus/mbppplus), and [DS-1000](https://github.com/HKUNLP/DS-1000/) for both completion (left-to-right) and insertion (FIM) mode are available.
+1.  **Set Environment Variables:**
+    Replace `{your_ip}` with the IP address of your deployed CodeGen service and `{your_model_identifier}` with the identifier of the model being tested (e.g., `Qwen/CodeQwen1.5-7B-Chat`).
 
-#### Environment
+    ```shell
+    export CODEGEN_ENDPOINT="http://{your_ip}:7778/v1/codegen"
```

Comment:
It's not consistent with Line 100. "codegen_url": "http://192.168.123.104:31234/v1/codegen", the port number is different.